sample/build.gradle

```diff
@@ -36,6 +36,10 @@ android {
     }
 }
 
+repositories {
+    maven { url 'https://jitpack.io' }
+}
+
```

Comment:
Can you add the repository in `/build.gradle` instead? Under `allprojects.repositories`.

Reply:
Actually, why do we need this repository? Can we just remove it?

Reply:
Correct, this can be removed entirely